packages/ui/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@friggframework/ui",
+  "private": true,
```

Comment:
This should switch to false or just be removed

Reply:
Need to add a sub-part of the package.json to make sure npm picks it up and publishes it... 
```
"publishConfig": {
    "access": "public"
  }
  ```